### DISCUSSION TITLE
时间胶囊

### DISCUSSION THREAD

**Discussion by ARMIN76L** [IntensiveCoLearning/Web3-URL#173] [Ideas and Project Teams]
因为一些私人事情可能不能完成接下来的共学，在此把自己项目的点子分享出来，有感兴趣的朋友可以尝试看看。
我们都知道一般在毕业季等时期大家可能会将对自己说的话、有纪念意义的物品等等一起放进一个大的容器然后埋起来，等待10/20年后回来聚在一起将这个容器打开。这种动不动10年20年后再打开的场景很适合搬到用ethstorage和web3url解决，大家再也不用“刨土”，不用担心多年以后回来胶囊被挖掘机铲走，合约提供access controll和人数、解封时间等等，上传文件的前端永远都在。

**Comment by luffythink** [comment]
非常棒的 idea 👍

**Comment by qizhou** [comment]
棒！

有一个技术问题是合约只提供写的access control，读还是公开的。所以怎么延迟解锁这个问题，除了区块链，还需要额外一些工具。
